python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/port_channel_interfaces.schema.yml

```diff
@@ -801,7 +801,10 @@ keys:
                 Hence, MPASS is needed only on MLAG port-channels connected to non-Arista devices.
         ip_address:
           type: str
-          description: IPv4 address/mask.
+          description: IPv4 address/mask or "dhcp".
```

Comment:
```suggestion
          description: IPv4 address/mask or `dhcp`.
```

Reply:
seems to be another one of these where we are inconsistent
I would keep it as @MaheshGSLAB did initially because we have the same for ethernet interfaces.